README.md

```diff
@@ -4,5 +4,13 @@ To run, follow these steps:
 
 1. Download ZIP or clone in GitHub
 2. Navigate to `liquid-gh-pages` folder or checkout `gh-pages` branch
-3. Run `bundle exec jekyll serve`
-4. Open `http://127.0.0.1:4000/liquid/` in your browser
+3. Run `bundle install` *(First time only. If it didn't work, see below.)*
+4. Run `bundle exec jekyll serve`
+5. Open [`http://127.0.0.1:4000/liquid/`](http://127.0.0.1:4000/liquid/) in your browser
+
+If you're on Windows and don't have `bundle`:
+
+1. If you don't have Ruby, install it *with Devkit* from [Ruby Downloads](https://rubyinstaller.org/downloads/).
+   * Be sure to run `ridk install` as the last step. Press *Enter* at the prompts.
```

Comment:
This seems like the wrong place to describe all the platform specific ways to install ruby.  Why not just add a step above getting started list for installing ruby that refer them to https://www.ruby-lang.org/en/downloads to describe how to  install ruby on different platforms.  It looks like that page already says "On Windows machines, you can use [RubyInstaller](https://rubyinstaller.org/)."

Reply:
That sounds better actually. But do we need to mention Devkit or `ridk install` in platform-agnostic instructions? I'm not sure if that's part of Linux/UNIX/macOS installs.

Reply:
> But do we need to mention Devkit or `ridk install` in platform-agnostic instructions?

No, I was saying it doesn't seem like it makes sense to mention it.  If they need to download ruby, then we can just refer them to https://www.ruby-lang.org/en/downloads that already mentions how RubyInstaller can be used to install ruby on windows.

Reply:
Ah sorry I miscommunicated. What I meant to say is that your suggestion sound better 😄 You're right, those Devkit and `ridk` probably apply on Windows only, not on other platforms. I'll take it out. Can you see the comment below, if that sounds right?